config/locales/en.yml

```diff
@@ -58,7 +58,7 @@ en:
       help: "Want to help?"
       help_ex: "diaspora* needs people to write and test code, welcome and help new members, set up and maintain community pods, and spread the word about the benefits of diaspora* to others who might want to take part. Think you’d like to contribute to the project?"
       socialint: "Social network integration"
-      socialint_ex: "Use diaspora* as your home base to post to your profiles on other major social services. This way your friends will still be able to keep in touch with what you’re up to, even if they’re not yet on diaspora*. diaspora* currently supports cross-posting to your Twitter, Tumblr, and WordPress accounts, with more to come."
+      socialint_ex: "Use diaspora* as your home base to post to your profiles on other major social services. This way your friends will still be able to keep in touch with what you’re up to, even if they’re not yet on diaspora*. diaspora* currently supports cross-posting to your Facebook, Twitter, and Tumblr accounts, with more to come."
```

Comment:
I think you added facebook back :)

Reply:
OK, I just applied https://github.com/diaspora/diaspora-project-site/commit/670ff1c75059b402a4a0fadb1867ab996afb0b6c again and squashed with your branch, and now it looks good :)